test/presubmit-tests.sh

```diff
@@ -59,6 +59,20 @@ function check_yaml_lint() {
 
 function ko_resolve() {
   header "Running `ko resolve`"
+
+  cat <<EOF > .ko.yaml
+    # This matches the value configured in .ko.yaml
```

Comment:
Do we have any guardrails in place to help keep these in sync? I can definitely imagine someone (me, probably) updating publish.yaml and not making the same update here.

This comment is a little misleading, since in this context it's actually matching values from publish.yaml, which is trying to match values in .ko.yaml

Reply:
Yeah, I just copy-pasted directly. And no, no guardrails - this is literally just the quick-and-dirty "lemme make sure this fails like I expect" approach for the moment.

Reply:
Oh, and actually, that comment is accurate - we're only overriding `baseImageOverrides`. But it's a confusing comment, so I'll remove it and the other down a few lines.

Reply:
Comment removed, added new comments here and in `tekton/publish.yaml` to remind people to keep them in sync.